gtfs-realtime/proto/gtfs-realtime.proto

```diff
@@ -906,6 +911,56 @@ message TranslatedString {
   extensions 9000 to 9999;
 }
 
+// An internationalized image containing per-language versions of a URL linking to an image
+// along with meta information
+// Only one of the images from a message will be retained by consumers. The resolution proceeds
+// as follows:
+// 1. If the UI language matches the language code of a translation,
+//    the first matching translation is picked.
+// 2. If a default UI language (e.g., English) matches the language code of a
+//    translation, the first matching translation is picked.
+// 3. If some translation has an unspecified language code, that translation is
+//    picked.
+// NOTE: This field is still experimental, and subject to change. It may be formally adopted in the future.
+message TranslatedImage {
+  message LocalizedImage {
+    // String containing a fully qualified URL linking to an image
+    // The image linked needs to be less than 500kB. 
```

Comment:
it's unclear whether the 500kb size limit is a recommendation or it is enforced?

Reply:
Yes my intent was that is enforced. I can change to SHOULD.

Reply:
>Yes my intent was that is enforced. I can change to SHOULD.

Do you mean it's a requirement? If so, it should be a MUST.